variables.tf

```diff
@@ -119,12 +119,11 @@ variable "node_pools" {
     # do not add nodecount because we enforce the use of auto-scaling
     max_count       = optional(number)
     min_count       = optional(number)
-    os_sku          = optional(string)
+    os_sku          = optional(string, "AzureLinux")
     mode            = optional(string)
     os_disk_size_gb = optional(number, null)
     tags            = optional(map(string), {})
     labels          = optional(map(string), {})
-    zones           = optional(set(string))
   }))
   default     = {}
```

Comment:
Question: if the default is `{}` the node pool will be created with Ubuntu that is the AKS default ?

Reply:
I have to test this, but if defaults to Ubuntu we have to change here:

https://github.com/Azure/terraform-azurerm-avm-ptn-aks-production/blob/4a690cb13328006940e2767c309447ddb49129e9/locals.tf#L42C37-L42C43

```
os_sku               = pool.os_sku
```

we can change to something like:

```
os_sku = coalesce(pool.os_sku, "AzureLinux")
```

Reply:
Forget about the above comments. I tested it and it is working as expected. If you dont specify `os_sku` in the object is going to use as default `AzureLinux` correctly. All good.